tests/e2e/runspecs.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+clear;
+#we need grab this value from docker container
+baseURL="http://docker.local:8088/ajax"
+
+
+entry="node node_modules/jasmine-node/bin/jasmine-node --noStack --config baseURL $baseURL "
+
+echo "Running all tests located in the e2e directory"
+command=$entry"spec/"
+echo $command
+time $command #/nested/uber-nested
+echo -e "\033[1;35m--- Should have 18 tests and 113 assertions and 0 Failure. ---\033[0m"
```

Comment:
why is this hardcoded? is there any reason we SHOULD NOT let frisby report its own results?

please fix this for all frisby tests!!!